prerendering.bs

```diff
@@ -641,6 +642,12 @@ This stores which client hints each origin has opted into receiving, until it ca
   1. Otherwise, [=map/set=] [=Accept-CH cache=][|origin|] to |hintSet|.
 </div>
 
+<h3 id="interaction-with-clear-site-data-clear-cache">Interaction with Clear-Site-Data Clear Cache</h3>
+
+Modify [$clear cache for origin$] to add the following step at the end.
```

Comment:
Wrap this whole thing in `<div algorithm="clear cache for origin patch">` `</div>`

Reply:
Looking from the other usages in the file for wrapping for patches, I assumed you meant the block in the other comment? Let me know if I need to change it, thanks!